operator/src/main/java/oracle/kubernetes/operator/helpers/ConfigMapHelper.java

```diff
@@ -137,6 +144,17 @@ static String extractFilename(String line) {
     return fname;
   }
 
+  /**
+   * getModelInImageSpecHash returns the hash for the fields that should be compared for changes.
+   *
+   * @return int hash value of the fields
+   */
+  public static int getModelInImageSpecHash(String imageName) {
+    return new HashCodeBuilder(17, 37)
+        .append(imageName)
```

Comment:
What about other data that is input to the domain home generation?  @jshum2479, remind me how this will be used.

Reply:
The restartVersion and introspectVersion has it own label in the pod, so the operator is already taking care of that.